src/config.js

```diff
@@ -28,8 +28,9 @@ const config = {
         useYou: true
       },
       realtimeActive: false,
+      omitCommentBubbles: true,
```

Comment:
Let's place it under `comments` section (to group them together) and rename to `omitRepeatedBubbles` (to avoid ambuguity). Also, they have to be off by default.

```
comments: {
  omitRepeatedBubbles: false
}
```